xen/arch/arm/coproc/coproc.c

```diff
@@ -183,9 +183,19 @@ static int coproc_attach_to_domain(struct domain *d,
         goto out;
     }
 
+    ret = iommu_assign_coproc(d, coproc->dev);
```

Comment:
so, this needs to be handled: either inside of iommu_assign_coproc (return OK if IOMMU doesn't need to be assigned) or here